templates/app/views/spree/user_passwords/edit.html.erb

```diff
@@ -1,7 +1,29 @@
 <article class="auth-container" id="change-password">
   <h1><%= I18n.t('spree.change_my_password') %></h1>
 
-  <%= render partial: 'spree/shared/error_messages', locals: { target: @spree_user } %>
+  <%= render 'spree/shared/error_messages', target: @spree_user %>
```

Comment:
Food for another PR but looks like a good candidate for a ViewComponent, WDYT?

Reply:
@kennyadsl Hmmm.. I would prefer to keep it as is because its logic is fairly simple. I think we would be making the partial more complicated if we wrapped it in a ViewComponent and/or we converted its html to `content_tag` blocks.

I would prefer prioritizing ViewComponent for complicated partials such as `templates/app/views/spree/components/search/_filter.html.erb`. 

Let me know if you have additional thoughts! :)

Reply:
The approach works for me, I'm sure you have more clear than me what we should prioritize!